concepts/classes/.meta/config.json

```
@@ -0,0 +1,5 @@
+{
+  "blurb": "JavaScript includes the capabilities for object-oriented programming. While it nowadays has a \"class\" keyword, it is still a prototype-based language.",
```

```suggestion
  "blurb": "JavaScript allows for object-oriented programming. Despite it having a \"class\" keyword, it is still a prototype-based language.",
```

Don't need to change this, but I would make it less wordy. This is an example :)